op-celestia/celestia/celestia_test.go

```diff
@@ -0,0 +1,35 @@
+package celestia
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestEncodeDecodeFrameRef(t *testing.T) {
```

Comment:
For these types of table tests, it is good to test valid and invalid test cases. 

So I would add a `isValid` field or something similar, so then in the test below, you can verify invalid cases do return an error.

Reply:
Added 👍🏽